tests/integration_tests/modules/test_ca_certs.py

```diff
@@ -105,59 +101,7 @@ def test_cert_installed(self, class_client: IntegrationInstance):
         )
 
     def test_clean_log(self, class_client: IntegrationInstance):
-        """Verify no errors, no deprecations and correct inactive modules in
-        log.
-        """
-        log = class_client.read_from_file("/var/log/cloud-init.log")
+        """Verify no errors or deprecations in log."""
+        log = class_client.execute("cat /var/log/cloud-init-output.log")
```

Comment:
why prefer `.execute("cat ..")` over `.read_from_file()`?

Reply:
I...have no idea why I did that :joy: